Update multi-cpu-architectures.md (#2564)
travis-ci/docs-travis-ci-com: user/multi-cpu-architectures.md

{: data-file=".travis.yml"}

You can also have a look at [Using Docker in Builds](user/docker/).

> ## Security and LXD container

Reviewer: ```suggestion
## Security and LXD Container
```

> Please note, that builds run in LXD containers will be denied access to privileged filesystems and paths. This is due to security reasons - a privileged container with write access to e.g. /sys/kernel/debugfs could create a messy situation on LXD host.

Reviewer: ```suggestion
> Due to security reasons, builds run in LXD containers will be denied access to privileged filesystems and paths - a privileged container with write access to e.g. /sys/kernel/debugfs might muddle an LXD host.
```

As a result, for instance a command in `.travis.yaml` like:
```yaml
sudo docker run --privileged --rm -t -v /sys/kernel/debug:/sys/kernel/debug:rw
```
{: data-file=".travis.yml"}

> would result in error.

Reviewer: ```suggestion
would result in an error.
```

> See e.g. [Github issue relevant to the topic](https://github.com/lxc/lxd/issues/2661) and [LXD apparmor setup](https://github.com/lxc/lxd/blob/master/lxd/apparmor/apparmor.go) for more details.

Reviewer: ```suggestion
Also have a look at the [Github issue relevant to the topic](https://github.com/lxc/lxd/issues/2661) and the [LXD apparmor setup](https://github.com/lxc/lxd/blob/master/lxd/apparmor/apparmor.go) for more details.
```
